app/assess/display_value_mappings.py

```diff
@@ -22,4 +22,5 @@
     "QA_COMPLETE": "QA Complete",
     "FLAGGED": "Flagged",
     "INELIGIBLE": "Ineligible",
+    "COMPLETED": "Completed",
```

Comment:
This should update the filter on the front-end Team dashboard to include "Completed"

This isn't in the prototype: https://fsd-pre-award.herokuapp.com/s26-assessment/dashboard

Although I don't see any issue with having it... will depend on design.  Also, if we just mark as completed, when does the QA related things take place? Is that done by different work in flags?

Reply:
If we end up leaving this in the front-end, can we make sure the search filter functionality actually works with the "Completed" status?  

I think it will judging by how the logic is in the assessment-store: https://github.com/communitiesuk/funding-service-design-assessment-store/blob/909ae0796a73ada165a23a1ea401b0cb2e3cb323/db/queries/assessment_records/queries.py#L60-L64

But worth checking in-case.  If you find a way to remove it from the UI to match the prototype, then we don't need to worry about it though.